lighty-core/dependency-versions/pom.xml

```diff
@@ -106,16 +106,25 @@
                 <artifactId>guice</artifactId>
                 <version>4.2.2</version>
             </dependency>
-            <dependency>
-                <groupId>org.testng</groupId>
-                <artifactId>testng</artifactId>
-                <version>6.14.3</version>
-                <scope>test</scope>
-            </dependency>
             <dependency>
                 <groupId>com.beust</groupId>
                 <artifactId>jcommander</artifactId>
                 <version>1.78</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
+                <version>4.5.13</version>
+            </dependency>
+            <dependency>
+                <groupId>io.fabric8</groupId>
+                <artifactId>kubernetes-client</artifactId>
```

Comment:
Why are we using this client? https://github.com/fabric8io/kubernetes-client vs https://github.com/kubernetes-client/java ?
This deserves an evaluation and explanation.

Reply:
Transition from fabric8 client to official one will be done in future changes